pybaseball/split_stats.py

```diff
@@ -0,0 +1,127 @@
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from datahelpers import postprocessing
```

Comment:
This needs to be `.datahelpers`, although it can also be removed I think

Reply:
This was removed. I'm good with the performance and realized that the issue I had earlier was on how I am handling this df after the split stats function. I'm good if you are.